util/linux.js

```diff
@@ -87,6 +87,11 @@ exports.gateway_ip_for = function(nic_name) {
   return trim_exec_async("ip r | grep " + nic_name + " | grep default | cut -d ' ' -f 3 | sed -n '1p'");
 };
 
+exports.bcast_address_for = function(nic_name) {
+  var cmd = "ifconfig " + nic_name + " 2> /dev/null | fgrep 'Bcast:' | awk '{print $3}' |awk -F: '{print $2}'";
```

Comment:
No need to do this in case output formay of ifconfig may vary on different platforms

Reply:
reverted as suggested